### DISCUSSION TITLE
Clone Error : Invalid Path due to File extension?

### DISCUSSION THREAD

**Discussion by NicholasOngIntel** [community/community#22664] [New to GitHub]
<p>I’m trying to clone a repo that isn’t owned by me and encountered the following error</p>
<p>Resolving deltas: 100% (43808/43808), done.<br>
error: invalid path 'ClientHealth_ClientHealthBenchmarkProcess_Cube_Migration.xml ’<br>
fatal: unable to checkout working tree<br>
warning: Clone succeeded, but checkout failed.<br>
You can inspect what was checked out with ‘git status’<br>
and retry with ‘git restore --source=HEAD :/’</p>
<p>Guessing that the file extension on the xml ends with a space and Windows doesn’t allow such a file to be created? The repo isn’t owned by me, so i can’t really edit the xml extension on the repo, is there anyway i can get around this? Thanks in advance</p>

**Comment by jcansdale** [comment, marked as answer]
<blockquote>
<p><a class="mention" href="/nicholasongintel">@nicholasongintel</a> wrote:</p>
</blockquote>
<blockquote>
<p>The repo isn’t owned by me, so i can’t really edit the xml extension on the repo, is there anyway i can get</p>
</blockquote>
<p>You might find the following useful:</p>
<p>Fixing Invalid Git Paths on Windows</p>
<p><a href="https://brendanforster.com/notes/fixing-invalid-git-paths-on-windows/" rel="nofollow noopener">https://brendanforster.com/notes/fixing-invalid-git-paths-on-windows/</a></p>
<p>The first part shows an easy way create a branch with the problem files removed.</p>
<p>I hope that helps!</p>

**Comment by wdtj** [comment]
<p>Note that if you are using sparse-checkout on windows, you may need to add core.protectNTFS false per <a href="https://github.com/git-for-windows/git/issues/2777" rel="nofollow noopener">https://github.com/git-for-windows/git/issues/2777</a></p>
